AlfrescoContent/AlfrescoContent/Classes/Swaggers/Models/SearchRequest.swift

```diff
@@ -22,18 +22,18 @@ public struct SearchRequest: Codable {
     public var defaults: RequestDefaults?
     public var localization: RequestLocalization?
     public var filterQueries: RequestFilterQueries?
-    public var facetQueries: RequestFacetQueries?
-    public var facetFields: RequestFacetFields?
-    public var facetIntervals: RequestFacetIntervals?
     public var pivots: [RequestPivot]?
     public var stats: [RequestStats]?
     public var spellcheck: RequestSpellcheck?
     public var scope: RequestScope?
     public var limits: RequestLimits?
     public var highlight: RequestHighlight?
     public var ranges: [RequestRange]?
+    public var facetQueries: [Queries]?
+    public var facetFields: Dictionary<String, [Fields]?>?
+    public var facetIntervals: Dictionary<String, [Intervals]?>?
```

Comment:
facet intervals have sets array, where are you passing it?

Reply:
These are in the Intervals array.